azure-pipelines-product-construction-service.yml

```diff
@@ -36,12 +37,12 @@ variables:
       value: -dev
 
 stages:
-- stage: PublishProductConstructionService
-  displayName: Publish Product Construction Service
+- stage: PublishPCS
```

Comment:
Nit: the stage is named `Publish*` but the display name is `Deploy *`. Same for the job